### DISCUSSION TITLE
have any documents about vueform+recapcha

### DISCUSSION THREAD

**Discussion by kvllua** [vueform/vueform#139] [Questions] [answered]
have any documents about vueform+recapcha

**Comment by adamberecz** [comment, marked as answer]
We don't have an official recaptcha element, however you're free to add it as a custom element. Here you can find more info on creating custom elements:
https://vueform.com/docs/creating-elements

Also implementing captcha is a planned feature, feel free to upvote:
https://github.com/vueform/vueform/discussions/67
